src/core/client/stream/mutations/CreateCommentMutation.ts

```diff
@@ -127,10 +157,18 @@ function commit(
       },
     } as any, // TODO: (cvle) generated types should contain one for the optimistic response.
     optimisticUpdater: store => {
+      // Skip if comments are always moderated.
+      if (alwaysPremoderated) {
+        return;
+      }
       sharedUpdater(environment, store, input);
       store.get(id)!.setValue(true, "pending");
     },
     updater: store => {
+      // Skip if comments are always moderated.
+      if (alwaysPremoderated) {
```

Comment:
Can you clarify what it means when the update is skipped when premod is enabled?

When premod is enabled for the story, there may be a situation in the future when a comment is automatically accepted/rejected, and I would rather your updater handle this more gracefully.

Reply:
That makes sense, I'll adapt it!

Reply:
Good catch @wyattjoh  😄